src/utils/searchAndSort.ts

```diff
@@ -74,11 +74,12 @@ const searchConditionMet = <T extends Record<string, unknown>>(result: T, condit
 
 // Try to get the `*(raw)` field if it exists, otherwise fall back to the regular field
 const getRawField = <T extends Record<string, unknown>>(result: T, field: string): unknown | undefined => {
-  const rawField = result[`${field}(raw)`];
-  if (rawField !== undefined) {
-    return rawField;
+  const fallback = [result[`${field}(raw)`], result[field], 0];
+  for (const value of fallback) {
```

Comment:
this looks good, will it affect the deliverables sort in any way?

Reply:
No, it should hit the second fallback, I'll add a test to illustrate. I think eventually the raw field will be implemented for deliverables, specifically for `numDocuments`.